pkg/asset/asset.go

```diff
@@ -56,6 +56,9 @@ const (
 	AssetPathSystemNamespace                = "manifests/kube-system-ns.yaml"
 	AssetPathCheckpointer                   = "manifests/pod-checkpointer.yaml"
 	AssetPathEtcdOperator                   = "manifests/etcd-operator.yaml"
+	AssetPathSelfHostedEtcdMemberPeerSecret = "manifests/etcd-member-peer-tls.yaml"
+	AssetPathSelfHostedEtcdMemberCliSecret  = "manifests/etcd-member-client-tls.yaml"
+	AssetPathSelfHostedOperatorEtcdSecret   = "manifests/operator-etcd-client-tls.yaml"
```

Comment:
Just to make sure we are consistent, this is (Etcd Operator)'s (Etcd Client)'s secret.

But change it to EtcdOperatorSecret is fine too.